x/valset/keeper/setup_test.go

```diff
@@ -24,12 +28,18 @@ type mockedServices struct {
 	SlashingKeeper *mocks.SlashingKeeper
 }
 
-func newValsetKeeper(t testing.TB) (*Keeper, mockedServices, sdk.Context) {
-	storeKey := sdk.NewKVStoreKey(types.StoreKey)
+func newValsetKeeper(t testing.TB) (*Keeper, mockedServices, context.Context) {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount("paloma", "pub")
+	config.SetBech32PrefixForValidator("palomavaloper", "valoperpub")
```

Comment:
use `CreateTestEnv`

Reply:
`CreateTestEnv` creates Testing environment but here we need only above config lines.So that is reason we have added here.

Reply:
Suggestion: let's move this definition to a custom function then. `SetupPalomaPrefixes` for example. It can then be used from within `CreateTestEnv`, but also called directly from here.